parodos-model-api/src/main/java/com/redhat/parodos/workflow/option/WorkFlowOptions.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (c) 2022 Red Hat Developer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.parodos.workflow.option;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * This consumed by a @see BaseAssessmentTask, updated by that task and then consumed by
+ * the UI layer for the user to choose which Workflow they wish to execute
+ *
+ * - currentVersion: The tool chain the application is currently using (it might be none -
+ * meaning tooling must be created)
+ *
+ * - upgradeOptions: Any available upgrade options for the existing tool chain
+ *
+ * - migrationOptions: Existing tooling/environments that the user can to move their
+ * workload over too (ie: moving from VMs to k8)
+ *
+ * - newOptions: New tool chains/environments that can be created for the application
+ *
+ * - continuationOptions: if a workflow has started and not been completed due to missing
+ * information from the user. Selecting one of these options allows them to continue
+ *
+ * - otherOptions: Workflows that are specific to the team (ie: adding/removing a
+ * developer from a project)
+ *
+ * @author Luke Shannon (Github: lshannon)
+ *
+ */
+public class WorkFlowOptions {
+
+	private WorkFlowOption currentVersion;
+
+	private List<WorkFlowOption> upgradeOptions;
+
+	private List<WorkFlowOption> migrationOptions;
+
+	private List<WorkFlowOption> newOptions;
+
+	private List<WorkFlowOption> continuationOptions;
+
+	private List<WorkFlowOption> otherOptions;
+
+	public boolean isOptionsAvailable() {
+		return !upgradeOptions.isEmpty() && !migrationOptions.isEmpty() && !newOptions.isEmpty()
+				&& !continuationOptions.isEmpty() && otherOptions.isEmpty();
+	}
+
+	public boolean hasInfrastructure() {
+		return currentVersion == null;
+	}
+
+	public boolean hasIncompleteWorkFlow() {
+		return !continuationOptions.isEmpty();
+	}
+
+	public WorkFlowOption getCurrentVersion() {
+		return currentVersion;
+	}
+
+	public void setCurrentVersion(WorkFlowOption currentVersion) {
+		this.currentVersion = currentVersion;
+	}
+
+	/**
+	 * Add a Continuation Options
+	 *
+	 */
+	public List<WorkFlowOption> getContinuationOptions() {
+		return continuationOptions;
+	}
+
+	public List<WorkFlowOption> addContinuationOption(WorkFlowOption continuationOption) {
+		upgradeOptions.add(continuationOption);
```

Comment:
Here we should use `continuationOptions`

Reply:
This is addressed